src/web/components/ClientSideTokenGeneration/CstgTable.tsx

```diff
@@ -49,8 +49,7 @@ export function CstgTable({
   const [pageNumber, setPageNumber] = useState<number>(initialPageNumber);
   const [rowsPerPage, setRowsPerPage] = useState<RowsPerPageValues>(initialRowsPerPage);
 
-  const isSelectedAll =
-    cstgValues.length && cstgValues.every((d) => selectedCstgValues.includes(d));
+  const isSelectedAll = selectedCstgValues.length === cstgValues.length;
```

Comment:
@ashleysmithTTD I'm pretty sure these are equivalent but this is more simple/performant, let me know if I'm missing something

Reply:
Yeah I think that's right!